barberia/src/App/pages/covertdate.js

```diff
@@ -0,0 +1,29 @@
+export function convertDate(str) {
+  var mnths = {
```

Comment:
change this to `const` instead of `var` cause you are never changing it.
also make the variable readable by calling it `months`

Reply:
fixed